Compilation error 'touch_pad_filter_start' was not declared in this scope

Hello,

I cloned the code
Entered the `ESP32LapTimer` folder
Then trying to upload:

```bash
pio run -e BOARD_OLD -t upload

src/Buttons.cpp: In function 'void newButtonSetup()':
src/Buttons.cpp:63:3: error: 'touch_pad_filter_start' was not declared in this scope
   touch_pad_filter_start(BUTTON1);
   ^~~~~~~~~~~~~~~~~~~~~~
src/Buttons.cpp:63:3: note: suggested alternative: 'touch_trigger_src_t'
Compiling .pio/build/BOARD_OLD/src/HardwareConfig.cpp.o
   touch_pad_filter_start(BUTTON1);
   ^~~~~~~~~~~~~~~~~~~~~~
   touch_trigger_src_t
src/Buttons.cpp:64:3: error: 'touch_pad_set_filter_period' was not declared in this scope
   touch_pad_set_filter_period(BUTTON1);
   ^~~~~~~~~~~~~~~~~~~~~~~~~~~
Compiling .pio/build/BOARD_OLD/src/Laptime.cpp.o
*** [.pio/build/BOARD_OLD/src/Buttons.cpp.o] Error 1

```

--- a/ESP32LapTimer/platformio.ini
+++ b/ESP32LapTimer/platformio.ini
@@ -9,7 +9,7 @@
 ; https://docs.platformio.org/page/projectconf.html
 
 [env]
-platform = espressif32
+platform = espressif32 @ ~3.5.0
 board = esp32dev
 framework = arduino
 monitor_speed = 115200

